File: libraries/core_legacy/src/main/java/com/tokopedia/core/network/entity/variant/ProductVariantResponse.java

package com.tokopedia.core.network.entity.variant;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ProductVariantResponse {

    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("data")
    @Expose
    private ProductVariant data;
    @SerializedName("server_process_time")
    @Expose
    private String serverProcessTime;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public ProductVariant getData() {
        return data;
    }

    public void setData(ProductVariant data) {
        this.data = data;
    }

    public String getServerProcessTime() {
        return serverProcessTime;
    }

    public void setServerProcessTime(String serverProcessTime) {
        this.serverProcessTime = serverProcessTime;
    }

}

File: features/merchant/product_add_edit/src/main/java/com/tokopedia/product/addedit/description/data/remote/model/variantbycat/ProductVariantResponse.kt
package com.tokopedia.product.addedit.description.data.remote.model.variantbycat

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by faisalramd on 2020-03-27.
 */

data class ProductVariantResponse(
        @SerializedName("data")
        @Expose
        val data: List<ProductVariantByCatModel> = emptyList(),

        @SerializedName("status")
        @Expose
        val meta: String = ""
)